kotlin/workflow-testing/src/test/java/com/squareup/workflow/testing/RenderTesterTest.kt

```diff
@@ -138,21 +140,19 @@ class RenderTesterTest {
   }
 
   @Test fun `getEventResult works`() {
-    val workflow = Workflow.stateful<Unit, String, String, (String) -> Unit>(
+    val workflow = Workflow.stateful<Unit, String, String, Sink<String>>(
         initialState = { _, _ -> fail() },
         render = { _, state ->
-          onEvent { event ->
-            enterState(
-                newState = "from $state on $event",
-                emittingOutput = "event: $event"
-            )
+          makeEventSink { event ->
+            this@makeEventSink.state = "from $state on $event"
```

Comment:
Yeah, I tried to avoid that in non-test code. Didn't care so much here.

Rationalization: at least it encourages you to break actions out of your `render` method, which usually seems to be a win.

Reply:
worth doing in the test, to set a good example?